core/src/com/mygdx/insanerpg/model/item/equipment/Equipment.java

```diff
@@ -5,4 +5,38 @@
 public abstract class Equipment extends Item {
     private int _maxDurability;
     private int _currentDurability;
+    private int _numTotalSockets;
+    private int _numFreeSockets;
```

Comment:
might want to have an int totalsockets and a list of sockets so you can just do the math instead of having a field for freesockets

Reply:
made an arrayList of Gem class